src/pages/Environments.js

```diff
@@ -456,8 +456,10 @@ const Environments = () => {
   // created, workspace namespace (a.k.a billing project) value used to equal the google project.
   // Therefore we use google project if the namespace label is not defined.
   const renderWorkspaceForApps = app => {
-    const { appType, googleProject, labels: { saturnWorkspaceNamespace = googleProject, saturnWorkspaceName } } = app
-    return getWorkspaceCell(saturnWorkspaceNamespace, saturnWorkspaceName, appType, false)
+    const { appType, googleProject, cloudContext: { cloudResource }, labels: { saturnWorkspaceNamespace = googleProject, saturnWorkspaceName } } = app
```

Comment:
can we get rid of `googleProject`

also shouldn't this be `{ appType, cloudContext: { cloudResource }, labels: { saturnWorkspaceNamespace = cloudResource, saturnWorkspaceName } }`?